test-cases/gutenberg/drag-and-drop-blocks.md

```diff
@@ -0,0 +1,369 @@
+# Drag and Drop Blocks - Test Cases
+
+##### TC001
+
+### Drag block to the top of the content
+
+**Preparation:**
+-   Create a new post/page.
+-   Tap on the `...` button to open the editor settings.
+-   Tap on the "Switch to HTML Mode" option.
+-   Copy and paste the [HTML example](./drag-and-drop-blocks.md#html-example) into the HTML editor.
+-   Tap again on the `...` button to open the editor settings.
+-   Tap on the "Switch to Visual Mode" option.
+
+**Steps:**
+-   Scroll down until reaching the middle of the block list.
+-   Long press over a block.
+-   Drag the block to the **top** of the screen (the block list will be automatically scrolled).
+-   Keep dragging the block until the block list reaches the top.
+-   Drop the block in the first position (a visual indication of where the block will be placed will be displayed).
+-   The block will be removed from its original position and inserted into the first position.
+
+<img src="../resources/drag-and-drop-blocks-drag-to-top-content.gif" width="200"/>
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### Drag block to the bottom of the content
+
+**Preparation:**
+-   Create a new post/page.
+-   Tap on the `...` button to open the editor settings.
+-   Tap on the "Switch to HTML Mode" option.
+-   Copy and paste the [HTML example](./drag-and-drop-blocks.md#html-example) into the HTML editor.
+-   Tap again on the `...` button to open the editor settings.
+-   Tap on the "Switch to Visual Mode" option.
+
+**Steps:**
+-   Scroll down until reaching the middle of the block list.
+-   Long press over a block.
+-   Drag the block to the **bottom** of the screen (the block list will be automatically scrolled).
+-   Keep dragging the block until the block list reaches the bottom.
+-   Drop the block in the last position (a visual indication of where the block will be placed will be displayed).
+-   The block will be removed from its original position and inserted into the last position.
+
+<img src="../resources/drag-and-drop-blocks-drag-to-bottom-content.gif" width="200"/>
+
+--------------------------------------------------------------------------------
+
+##### HTML example
+The following HTML is an example to be used for testing the drag and drop blocks functionality.
+<details><summary>Click here to display the HTML</summary>
```

Comment:
I wonder if it'd be nice to put a QR code here with the link and the anchor. So it's easy to open this from the device to be able to copy it. What do you think? Just thinking when we actually have to follow these steps 😁

Reply:
Yeah, that's a good idea ❤️. It will definitively help in following the step, I'll add a QR code image pointing to the link.

Reply:
I added your suggestion in https://github.com/wordpress-mobile/test-cases/pull/118/commits/65952b84ef99567ed38666285b82435b111a7943.